torch/fx/experimental/symbolic_shapes.py

```diff
@@ -231,8 +232,7 @@ def _make_magic(method, func, py_type):
 
     def magic_impl(self, other):
         if method in ["min", "max"]:
-            # op = getattr(builtins, method)
-            return self
+            op = getattr(builtins, method)
```

Comment:
@albanD I could delete this PR, but I noticed this - probably still worth landing?